src/main/resources/lang/zh_cn/messages.yml

```
@@ -531,4 +533,34 @@ FREE_LICENSE_RENEW_SUCCESS: "新的许可证已生成并激活"
+MATCH_STRING_EQUALS: "字符串匹配: {}"
+MATCH_STRING_LENGTH: "字符串长度: {}"
+MATCH_STRING_REGEX: "字符串正则表达式: {}"
+MODULE_IBL_COMMENT_UNKNOWN: "未提供"
+JSON_MATCHER_NOT_MET: "表达式未满足: {} 于 {} 上"
+DOWNLOADER_API_TEST_BYPASS_PAUSED: "下载器配置已暂存，请在稍后启用"
+STATUS_BAR_GLOBAL_PAUSED: "⏸全局暂停"
+BTN_CONFIG_STATUS_UNSUCCESSFUL_HTTP_REQUEST: "向 URL {} 发送 HTTP 请求失败，HTTP 状态码: {}，远端响应：{}"
+BTN_CONFIG_STATUS_UNSUCCESSFUL_INCOMPATIBLE_BTN_PROTOCOL_VERSION_CLIENT: "当前客户端 BTN 协议实现版本过旧，本地实现版本号: {}，服务器要求最低: {}"
+BTN_CONFIG_STATUS_UNSUCCESSFUL_INCOMPATIBLE_BTN_PROTOCOL_VERSION_SERVER: "当前服务器 BTN 协议实现版本过旧，服务器实现版本号: {}，本地要求最低: {}"
+BTN_CONFIG_STATUS_SUCCESSFUL: "已成功连接到 BTN 服务器"
+BTN_CONFIG_STATUS_EXCEPTION: "配置 BTN 功能时发生意外错误：{}: {}"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)